pmg/templates/committee_detail.html

```diff
@@ -92,16 +117,14 @@ <h3 class="collapse-link"><span class="fa fa-caret-right"></span> <a href="#">Ab
               {% endfor %}
             </table>
 
-            {% if committee.calls_for_comments|length > 5 %}
+            {% if calls_for_comments|length > 5 %}
               <a href="/calls-for-comments/?filter[committee]={{ committee.id }}" class="btn btn-primary">More</a>
             {% endif %}
-          </div>
-        {% endif %}
-
-        {% if recent_questions %}
-          <div class="col-md-6">
-            <h3><span class="fa fa-question-circle"></span> Questions and Replies</h3>
-            <table class="table table-compact">
+          {% endif %}
+        </div>
+        <div class="tab-pane fade in c-m-q-replies" id="qr">
```

Comment:
They need the id in order to be targeted by bootstrap tabs. The class is there in case any additional styling changes need to be made.